### PR TITLE
[Known Issue] Add docs to describe a known issue/limitation of EQL rule cross-cluster search 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Contributes to https://github.com/elastic/security-docs/issues/4744

Previews:
- [Create a detection rule | Create an event correlation rule](https://security-docs_bk_4804.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#create-eql-rule)
- [Create rule | Optional fields for event correlation, query, threshold, indicator match, new terms, and ES|QL rules](https://security-docs_bk_4804.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html#opt-fields-query-threshold)

### CODE REVIEW

File: docs/detections/api/rules/rules-api-create.asciidoc [hunk 1]
```diff
@@ -424,7 +424,12 @@ Security Solution indices defined on the {kib} Advanced Settings page
 (*Kibana* → *Stack Management* → *Advanced Settings* →
 `securitySolution:defaultIndex`).
 
-NOTE: This field is not supported for ES\|QL rules.
+[NOTE]
+======
+
```
Comment (joepeeples) [inline docs/detections/api/rules/rules-api-create.asciidoc:429]: Remove empty line?

Comment (nastasha-solomon) [reply docs/detections/api/rules/rules-api-create.asciidoc:429]:
```suggestion
```

File: docs/detections/api/rules/rules-api-create.asciidoc [hunk 2]
```diff
@@ -424,7 +424,12 @@ Security Solution indices defined on the {kib} Advanced Settings page
...
+- This field is not supported for ES\|QL rules.
+- Event correlation rules have a limitation that prevents them from querying multiple indices from different clusters (local and remote). To workaround this known issue, a user with the {ref}/built-in-roles.html[`superuser`] role must modify the EQL rules that are configured to use <<rules-cross-cluster-search,cross-cluster search>>. Doing this updates the rule's API key to use `superuser` privileges and allows the rule to use cross-cluster search.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/api/rules/rules-api-create.asciidoc:430]: This works, ty!

File: docs/detections/rules-ui-create.asciidoc [hunk 3]
```diff
@@ -137,6 +137,9 @@ For example, if a rule runs every 5 minutes but you don't need alerts that frequ
 . Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
 . To create an event correlation rule using EQL, select *Event Correlation*, then:
 .. Define which {es} indices or data view the rule searches when querying for events.
++
+NOTE: Event correlation rules have a limitation that prevents them from querying multiple indices from different clusters (local and remote). To workaround this known issue, a user with the {ref}/built-in-roles.html[`superuser`] role must modify the EQL rules that are configured to use <<rules-cross-cluster-search,cross-cluster search>>. Doing this updates the rule's API key to use `superuser` privileges and allows the rule to use cross-cluster search.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:141]:
```suggestion
NOTE: Event correlation rules have a limitation that prevents them from querying multiple indices from different clusters (local and remote). To enable this, a user with the {ref}/built-in-roles.html[`superuser`] role must modify the EQL rules that are configured to use <<rules-cross-cluster-search,cross-cluster search>>. This updates the rule's API key to use `superuser` privileges and allows the rule to use cross-cluster search.
```

**Review by joepeeples — APPROVED** [review]

**Comment by e40pud** [comment]
@nastasha-solomon I see `8.12.0` label, does that mean that we plan to add these docs to that version as well? The issue happens in version prior to `8.12.0` and fixed in it.

**Comment by nastasha-solomon** [comment]
Thanks for the reminder, @e40pud! I corrected this in https://github.com/elastic/security-docs/pull/4813 so the doc updates will only be applied to the 7.17-8.11 docs.

**Pull request closed without merging**